cypress/e2e/landing.test.ts

```diff
@@ -40,16 +40,23 @@ describe('Landing Page', () => {
     cy.url().should('include', '/pools')
   })
 
-  it('does not render landing page in when path is blocked', () => {
-    cy.visit('/', { userState: DISCONNECTED_WALLET_USER_STATE })
-    cy.document().then((doc) => {
-      const meta = document.createElement('meta')
-      meta.setAttribute('property', 'x:blocked-paths')
-      meta.setAttribute('content', '/,/nfts,/buy')
-      doc.head.appendChild(meta)
+  it('does not render landing page when / path is blocked', () => {
+    cy.intercept('/', (req) => {
+      req.reply((res) => {
+        const parser = new DOMParser()
+        const doc = parser.parseFromString(res.body, 'text/html')
+        const meta = document.createElement('meta')
+        meta.setAttribute('property', 'x:blocked-paths')
+        meta.setAttribute('content', '/,/buy')
+        doc.head.appendChild(meta)
+
+        res.body = doc.documentElement.outerHTML
+      })
     })
```

Comment:
nit: this would be a good candidate for a cypress test utility function, as it is re-used across tests. not necessary, though